src/pages/Admin/Room/Modal/Item/Item.tsx

```diff
@@ -115,17 +116,14 @@ const RoomModalItem: React.FC<RoomModalItemProps> = ({
         <h4 className="italic input-header">Create venue</h4>
       </S.Flex>
       <S.Flex>
-        <label id={"useUrl"} className="switch">
-          <input
-            type="checkbox"
-            id={"useUrl"}
-            name={"useUrl"}
-            checked={useUrl}
-            onChange={handleUrlToggle}
-            ref={register}
-          />
-          <span className="slider round"></span>
-        </label>
+        <Toggler
+          labelClassName="switch"
```

Comment:
same with the label

Reply:
removed `labelClassName` 7983b12912a95f067c32c1fc6ec8d47716e372d0